README.md

```diff
@@ -21,6 +21,19 @@ which riscv32-unknown-elf-gcc
 ```
 is not empty.
 
+## LLVM Toolchain
+
+The applications are built using the RISC-V LLVM toolchain. On IIS systems, users can use the pre-installed LLVM version 12.0.1.
+
+Outside of IIS systems, you need to install LLVM version 12.0.1 or later to ensure compatibility.
+
+The correct version of the toolchain can be verified by running
+```
+llvm-config --version
+```
```

Comment:
See the other comment. Then you could then specify the toolchain location via the `TOOLCHAIN_DIR` variable.

Reply:
And how do I  specify to use the preinstalled LLVM version 12.0.1 on the IIS machines?